source/lex.tex

```diff
@@ -1136,11 +1136,11 @@
 \indextext{prefix!\idxcode{u8}}%
 is a character literal of type \tcode{char},
 known as a \defn{UTF-8 character literal}.
-The value of a UTF-8 character literal
+The value of a UTF-8 character literal containing a single \grammarterm{c-char}
 is equal to its ISO 10646 code point value,
 provided that the code point value
-is representable with a single UTF-8 code unit
-(that is, provided it is in the C0 Controls and Basic Latin Unicode block).
+is representable with a single UTF-8 code unit.
+(That is, provided it is in the C0 Controls and Basic Latin Unicode block.)
```

Comment:
I would have too, but I chose this way because the `char16_t` wording has been around for longer. I can change it to the other version for both if other people prefer it too.

I didn’t see in the [How to submit an editorial issue](https://github.com/cplusplus/draft/wiki/How-to-submit-an-editorial-issue) page whether you’d prefer an amended commit or a separate one; I would guess amended, and that I should also rebase while I’m at it?